docker.go

```diff
@@ -1369,6 +1369,16 @@ func (p *DockerProvider) ReuseOrCreateContainer(ctx context.Context, req Contain
 		lifecycleHooks:    []ContainerLifecycleHooks{combineContainerHooks(defaultHooks, req.LifecycleHooks)},
 	}
 
+	if p.config.RyukDisabled {
+		// ryuk disabled, ensure the container we are about to reuse is started
+		if !dc.IsRunning() {
```

Comment:
Bug: `dc.IsRunning` always returns false, that struct was initialized right above... :facepalm: 
```suggestion
		if c.State == "running" {
```